roofit/roofitcore/src/NumericalDerivatorMinuit2.cxx

```diff
@@ -84,49 +84,31 @@ NumericalDerivatorMinuit2::~NumericalDerivatorMinuit2()
 void NumericalDerivatorMinuit2::setup_differentiate(const ROOT::Math::IBaseFunctionMultiDim *function, const double *cx,
                                                     const std::vector<ROOT::Fit::ParameterSettings> &parameters)
 {
-
    assert(function != nullptr && "function is a nullptr");
 
-   auto get_time = []() {
-      return std::chrono::duration_cast<std::chrono::nanoseconds>(
-                std::chrono::high_resolution_clock::now().time_since_epoch())
-         .count();
-   };
-   decltype(get_time()) t1, t2, t3, t4, t5, t6, t7, t8;
-
-   t1 = get_time();
```

Comment:
Can this commit be squashed with the place where the timing code is introduced, so it's a no-op?